data/examples/declaration/data/field-layout/record-1-out.hs

```diff
@@ -0,0 +1,11 @@
+-- | Foo.
+data Foo
+  = Foo
+  { -- | Something
+    foo :: Foo Int Int,
+    -- | Something else
+    bar ::
+      Bar
+        Char
+        Char
+  }
```

Comment:
Hmm, the `{}` brackets enclosing the fields are at the same indentation level as the `=`, but if you add another constructor in the end, then they are indented by two more steps:
```haskell
data Foo
  = Foo
      { -- | Something
        foo :: Foo Int Int,
        -- | Something else
        bar ::
          Bar
            Char
            Char
      }
  | Bar
```
This feels a bit inconsistent, but also not like a huge deal. WDYT?

Reply:
Yeah, that was a decision that I made a while ago. A single constructor records are the default so it made sense to save a little bit of horizontal space by not aligning them. This PR simply keeps the status quo.